exampleSite/static/admin/config.yml

```diff
@@ -307,23 +307,46 @@ collections:
         widget: "object"
         fields:
           - {label: "Discord Name", name: "discord_id", widget: "string"}
-  # encounter metadata
-  - label: "Encounter metadata"
-    name: "encounter-guide"
-    folder: "data/encounter"
-    format: "json"
-    identifier_field: "category"
+  # Encounters Metadata
+  - label: "Encounters"
+    name: "encounter-data"
+    folder: "content/encounters"
+    identifier_field: "fight_title"
     create: true
     fields:
-      - {label: "Category", name: "category", widget: "string"}
-      - label: "Encounters" 
-        name: "encounters"
+      - {label: "Fight Title", name: "fight_title", widget: "string"}
```

Comment:
I'd add a title field back above this, so we can use `.Title` elsewhere. 

Also, for this widget, might be best if we add a description *and* validation, so probably break this out like this:

```yml
fields:
  - label: Fight Title
    name: fight_title
    description: This is the short name of the fight, like "o12s" or "e12s-p2"
    widget: string
    pattern: ["[A-Za-z0-9\-]", "Cannot use spaces. Please only use letters, numbers, and dashes"]
```